server/chat-commands/core.ts

```diff
@@ -370,7 +370,7 @@ export const commands: ChatCommands = {
 		if (this.cmd === 'mee' && /[A-Z-a-z0-9/]/.test(target.charAt(0))) {
 			return this.errorReply(this.tr`/mee - must not start with a letter or number`);
 		}
-		this.checkChat(`/${this.cmd} ${target || ''}`);
+		target = this.checkChat(`/${this.cmd} ${target || ''}`);
```

Comment:
This is probably a mistake.

Reply:
I don't think so? Without this, `/me` doesn't work.

Reply:
I'm fixing this in #7327. You can remove it.